chickadee/processes/wps_BCCAQ.py

```diff
@@ -58,15 +73,9 @@ def __init__(self):
         )
 
     def _handler(self, request, response):
-        (
-            gcm_file,
-            obs_file,
-            varname,
-            out_file,
-            num_cores,
-            end_date,
-            loglevel,
-        ) = [arg[0] for arg in collect_args(request, self.workdir).values()]
+        args = [arg[0] for arg in collect_args(request, self.workdir).values()]
+        (gcm_file, obs_file, varname, out_file, num_cores, loglevel) = args[:6]
```

Comment:
Is there a reason we cannot assign the args here without the index?

Reply:
I enabled it to deal with the cases where there are multiple arguments per input. That's why we need indexing.